safe/safe.go

```diff
@@ -0,0 +1,21 @@
+package safe
+
+import (
+	"log"
+	"runtime/debug"
+)
+
+// GoSafe starts a recoverable goroutine
+func GoSafe(goroutine func()) {
```

Comment:
I think the name could be `Go` and that's all ; that way, no repeat of the `safe` word, and it looks cool `safe.Go(…)`.